react/Modal/Readme.md

```diff
@@ -7,7 +7,8 @@ initialState = { modalDisplayed: false};
   <button onClick={()=>setState({ modalDisplayed: !state.modalDisplayed })}>
     Toggle modal
   </button>
-  {state.modalDisplayed && <Modal title='Ada Lovelace' description={content.ada.short} dismissAction={() => setState({ modalDisplayed: false })} /> }
+  {state.modalDisplayed && <Modal into="#modalContainer" title='Ada Lovelace' description={content.ada.short} dismissAction={() => setState({ modalDisplayed: false })} /> }
+  <div id="modalContainer" />
```

Comment:
You can't append the modal directly into the `body` anymore? :'(

Reply:
Yes you can with `into="body"` you can see [here](https://github.com/cozy/cozy-ui/pull/594/files#diff-113f5e7400f32829864d3c8c2c2fe97eR4).

Reply:
Oh ok, the example misled me, thanks.